cmd/rancher-desktop-guestagent/main.go

```diff
@@ -52,6 +52,7 @@ var (
 		"file path for Containerd socket address")
 	vtunnelAddr             = flag.String("vtunnelAddr", vtunnelPeerAddr, "Peer address for Vtunnel in IP:PORT format")
 	enablePrivilegedService = flag.Bool("privilegedService", false, "enable Privileged Service mode")
+	listenerAddr            = flag.String("listenerAddr", net.IPv4zero.String(), "default service listener address")
```

Comment:
Also, it would be better if the description included a little more detail on what it's for. Perhaps, something like:
`Address to bind Kubernetes services to on the host, available options are 0.0.0.0 or 127.0.0.1`.

Reply:
Yes, thank you for the suggestion :+1: